conduit/plugins/importers/syncmode/syncmode_importer.go

```diff
@@ -0,0 +1,168 @@
+package syncmode
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+
+	"github.com/algorand/indexer/conduit"
+	"github.com/algorand/indexer/conduit/plugins"
+	"github.com/algorand/indexer/conduit/plugins/importers"
+	"github.com/algorand/indexer/data"
+
+	"github.com/algorand/go-algorand-sdk/v2/client/v2/algod"
+	sdk "github.com/algorand/go-algorand-sdk/v2/types"
+	"github.com/algorand/go-algorand/protocol"
+	"github.com/algorand/go-algorand/rpcs"
+)
+
+const importerName = "sync-algod"
+
+type syncModeImporter struct {
+	aclient *algod.Client
+	logger  *logrus.Logger
+	cfg     Config
+	ctx     context.Context
+	cancel  context.CancelFunc
+}
+
+func (sm *syncModeImporter) OnComplete(input data.BlockData) error {
+	_, err := sm.aclient.SetSyncRound(input.Round() + 1).Do(sm.ctx)
+	return err
+}
+
+//go:embed sample.yaml
+var sampleConfig string
+
+var syncModeImporterMetadata = conduit.Metadata{
+	Name:         importerName,
+	Description:  "Importer for fetching blocks from an algod REST API.",
```

Comment:
this description is the same as algod_importer. does anyone else have a better description for this plugin?

Reply:
I changed it a little.  Let me know what you think.  FYI @Eric-Warehime